.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!-- Please fill out ALL information here, and don't mess with the formatting. -->
+<!-- HTTPS Everywhere uses a bot to manage issues and if you change things, it won't be able to understand. -->
+
+<!-- Delete all but one of the next five lines to indicate the type of issue you're filing. -->
+Type: ruleset issue
+Type: new ruleset
+Type: code issue
+Type: feature request
+Type: other
+
+<!-- If you're reporting a ruleset issue, fill out this section. -->
+Domain:
```

Comment:
You should describe how to specify multiple domains.  Should they be comma-separated?  What about subdomains?